packages/app/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,35 @@
+"use client";
+import Image from "next/image";
+import Blocks from "../../public/assets/images/blocks.svg";
+import Link from "next/link";
+import MobileMenu from "./MobileMenu";
+import Logo from "./Logo";
+
+export default function Navbar() {
+  return (
+    <header className="h-[72px] top-0 flex flex-col w-full bg-surface-25 border-b border-solid border-b-surface-75">
```

Comment:
IMO `72px` should be a constant shared between `Navbar` and `MobileMenuList`

Reply:
Moved to tailwind extend config.